build/jest-transformer.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-env node */
+
+const transformer = require('babel-jest').createTransformer({
+  presets: [
+    'babel-preset-flow',
+    'babel-preset-es2015',
```

Comment:
Is the es2015 preset required given that we require Node 8+?

Reply:
I'll try getting rid of it. 👍

Reply:
Turns out this is required for modules to work. Might be able to do this with a smaller plugin, but since it's only for tests, I don't think it's too bad.